lockbox-ios/FxA/FxAView.swift

```diff
@@ -8,34 +8,46 @@ import RxSwift
 
 class FxAView : UIViewController, FxAViewProtocol, WKNavigationDelegate {
     internal var presenter: FxAPresenter!
-    private var webView: WKWebView
+    private var webView: WKWebView!
     private var disposeBag = DisposeBag()
 
     init(webView: WKWebView? = WKWebView()) {
```

Comment:
Can this argument not be optional but have a default value? So instead have it `init(webView: WKWebView = WKWebView()) {` That way we don't need the force upwrap above.

Reply:
good cleanup note, thanks.